Homework_4_11_If_Number_4/Homework_4_11_If_Number_4/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework_4_11_If_Number_4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Для вычисления значения функции f(x) введите значение аргумента х");
+            var x = double.Parse(Console.ReadLine());
+            Function(x);
+
+            Console.ReadKey();
+        }
+
+        static void Function(double x)
+        {
+
+            if (x > 2)
+            {
+                Console.WriteLine("f(x) = 2");
+            }
+            else if (x >= -2 & x <= 2)
```

Comment:
Лишнее логическое условие. В эту ветку мы попадем только , если _x_ <= 2
```suggestion
            else if (x >= -2)
```